internal_workflow_testsuite_test.go

```diff
@@ -77,6 +77,7 @@ func (s *WorkflowTestSuiteUnitTest) Test_ActivityMockFunction() {
 
 func (s *WorkflowTestSuiteUnitTest) Test_ActivityMockValues() {
 	env := s.NewTestWorkflowEnvironment()
+	env.SetActivityTaskList(defaultTestTaskList, RegisterActivityOptions{}, testActivityHello)
```

Comment:
This was due to a change in behavior causing testWorkflowEnv not to work properly. Fixed now.